### DISCUSSION TITLE
DJA to be part of Hacktoberfest

### DISCUSSION THREAD

**Discussion by sliverc** [django-json-api/django-rest-framework-json-api#983] [General]
Maybe you are already aware of [Hacktoberfest](https://hacktoberfest.digitalocean.com/) maybe you are not. I have been joining this initiative for a couple of years already. It is an initiative that during the month of October if you get 4 pull requests approved on any open source projects that joins this initiative you get a free T-Shirt... Yeay. For more information and how to sign up read more on their [website](https://hacktoberfest.digitalocean.com/).

I have also marked this project to join Hacktoberfest. So any PR created and approved during the month of October counts towards your Hacktoberfest counter. I hope to see the PRs increasing during this month... :smile:

To get started working on Django REST framework JSON:API look [here](https://github.com/django-json-api/django-rest-framework-json-api/contribute) for some easy picks. But feel free to choose from any of the DJA [issues](https://github.com/django-json-api/django-rest-framework-json-api/issues), we need help with them all.

**Comment by auvipy** [comment]
I have one PR open so let me complete and get that merged in october :D

**Reply by sliverc** [reply]
From the [Hacktoberfest FAQ](https://hacktoberfest.digitalocean.com/faq):

> Pull requests created before Oct 1 but merged or marked as ready for review after do not count.

**Comment by sliverc** [comment]
Almost missed it this year, but Hacktoberfest 2022 is on right now and we also participate as DJA (flagged topic hacktoberfest). Description of initial discussion still applies.
